### PR TITLE
create_other_files

### PR DESCRIPTION
**Pull request by kennedyowusu**
Created and added HTML code in the index file

Created CSS file and linked to it in the index.html file

Added the linters configuration

### CODE REVIEW

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @kennedyowusu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
- [ ] I cannot leave a comment there so I am leaving it here please implement all the required changes.
- [ ] I would suggest you please wrap the `logo` with an `a` tag for semantics tag, and for better user experience(UX).
- [ ] I would suggest you please wrap your menu with a `nav` tag because in the next milestone you will add more nav links.
- [ ] kindly use `hx` tag for you sub-title so replace `p` with `hx` like `h2`,`h3`,`h4` for better semantic HTML.
- [ ] when I decrease the size then I found the spacing on the right side. please fix it. There is more spacing after the content fix it as well and please remove the list-style to none as there is `.` on every icon.
- [ ] The paragraph content looks different so use the `font-family: 'Poppins` and fix font-weight and stick as much as possible to the design
### your design
his
### Figma design
figma
- [ ] I would suggest you please use the same font provided by microverse ie: `font-family: 'Poppins`


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @kennedyowusu 👋 
Great job on implementing most of the request changes 👏 There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
- [ ] The white spacing that the previous reviewer mentioned is still happening. Kindly review your CSS and fix the issue 👍 

Animation

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kennedyowusu** [comment]
The error has been rectified. One thing that I am struggling to understand is, what screen size has a max-width below 350px? I must register my frustration and near impossibility in adjusting the screen to fit a screen below the max-width of 350px.

**Comment by kennedyowusu** [comment]
##  Feedback Implemented

I have implemented the feedback. I have even made sure that a screen with a max-width of even 260px can still have a nicer look. Checking from the BBC site, a screen with a max-width of 30px even looks distorted. How I am expected to perfect that still baffles me.

**Comment by Nyame-Wolf** [comment]
Hi @kennedyowusu  please note that the minimum screen size is 375 px.

File: index.html [hunk 1]
```diff
@@ -10,17 +10,25 @@
...
     </header>
 
     <main>
       <section class="top-section">
         <h1 class="title">Hey There. I'm Kennedy</h1>
-        <p class="sub-title">I'm a software developer</p>
+        <h4 class="sub-title">I'm a software developer</h4>
         <p class="description">
           I can help you build a product , feature or website. Look through some
           of my work and experience! If you like what you see and have a project
```
Comment (Nyame-Wolf) [inline index.html:34]: Kindly note that this sections background image is not at the bottom right position 

- this is your design
Screenshot 2022-09-23 at 08 50 20

- this is the figma design
Screenshot 2022-09-23 at 08 53 42

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @kennedyowusu  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great work using the right GitHub Flow :100: 
- Great work using the same font as Figma design ☑️ 
- Commits are descriptive and meaningful :+1: 
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

- [ ] Kindly note that you need a more descriptive branch name. A branch name should tell you what exact feature you are building. instead of `create_other_files` you can use `mobile_setup`

 _Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by kennedyowusu** [comment]
## Hi @Nyame-Wolf

I have implemented the recommendations you asked for and created a new branch called mobile_setup. Kindly check it out and give me feedback before I proceed to request for review.

Thank you.

**Pull request merged**